modules/tinymce/src/plugins/lists/main/ts/actions/ToggleList.ts

```diff
@@ -316,17 +316,34 @@ const updateList = (editor: Editor, list: Element, listName: 'UL' | 'OL' | 'DL',
   }
 };
 
+const updateCustomList = (editor: Editor, list: Element, listName: 'UL' | 'OL' | 'DL', detail: ListDetail): void => {
+  list.className = list.className.split(' ').filter((cls) => !/\btox\-/.test(cls)).join(' ');
```

Comment:
Surely there's a nicer way to do this by iterating over the `classList` rather than split+filter+join+trim 🤔 
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

- [ ] update both places

Reply:
Done!